### DISCUSSION TITLE
I18n for annotations

### DISCUSSION THREAD

**Discussion by fdw** [fdw/rofimoji#109] [Q&A]
Unicode publishes translated annotations for many characters. Now I'm wondering if that is something anyone would be interested in... the main description would still be English, but the key words in parentheses could be from any language (or several).
The main drawback is some complexity, but mainly the size of all the annotations - adding 10 MB to a project that had previously not even half a MB.
Opinions?

**Comment by PirMei** [comment]
I came here from https://github.com/fdw/rofimoji/issues/200 and while I personally would love a way to control which officially translated annotations should be used for finding emojis (in my case English and German), I would at least suggest adding [your example line](https://github.com/fdw/rofimoji/issues/200#issuecomment-2351709672) for additional emoji annotations in the README, section `Custom character files and descriptions`.

**Reply by fdw** [reply]
I've expanded the documentation a bit; hope that helps.

Regarding the full i18n, I'm afraid there's not a lot of demand for it. But I'll keep this discussion open and wait 🙂
